File: MyungJi/231120_week/programmers_161990_바탕화면정리.cpp
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    vector<int> answer = {100, 100, -100, -100};
    for(int i=0;i<wallpaper.size();i++){
        for(int j=0;j<wallpaper[0].size();j++){
            if (wallpaper[i][j]=='#'){
                answer[0] = min(answer[0], i);
                answer[1] = min(answer[1], j);
                answer[2] = max(answer[2], i+1);
                answer[3] = max(answer[3], j+1);
            }
            
        }
    }

    return answer;
}

File: MyungJi/240304_week/programmers_161990_바탕화면정리.java
import java.util.ArrayList;

class Solution {
    public int[] solution(String[] wallpaper) {
        ArrayList<Integer> answer = new ArrayList<>();
        int maxNum = 54;
        int lux = maxNum, luy = maxNum, rdx = 0, rdy = 0;
        for (int i=0;i<wallpaper.length;i++){
            for (int j=0;j<wallpaper[i].length();j++){
                if (wallpaper[i].charAt(j) == '#'){
                    lux = Math.min(i, lux);
                    luy = Math.min(j, luy);
                    rdx = Math.max(i, rdx);
                    rdy = Math.max(j, rdy);
                }
            }
        }
        answer.add(lux);
        answer.add(luy);
        answer.add(rdx+1);
        answer.add(rdy+1);

        return answer.stream().mapToInt(x->x).toArray();
    }
}